### PR TITLE
Implement Classroom, Student, Book, and Rental Classes with Relationships

### PR DESCRIPTION
**Pull request by peter-akhigbe**
This pull request adds the implementation for the Classroom, Student, Book, and Rental classes as per the project requirements. The Classroom class has a one-to-many relationship with the Student class, and the Person and Book classes have a many-to-many relationship through the Rental class.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require './namable'
+require './rental'
 require 'securerandom'
 
 class Person < Namable
```
Comment (cynthiainga) [inline person.rb:5]: - [ ] `Getters` and `Setters` for `@rentals` should be present, please kindly add it 🙏🏾 
- [ ] Also `@rentals` should be initialized, you didn’t initialize it kindly fix that 🙃

File: classroom.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
```
Comment (cynthiainga) [inline classroom.rb:2]: - [ ] The `@students` should be a `attr_reader`, please kindly fix that 🙏🏾

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @peter-akhigbe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] The `has-many` / `belongs-to` relationships between `Classroom` and `Student` are created ✔️ 
- [ ] A book has many rentals ✔️ 
- [ ] A rental belongs to a book and also belongs to a person ✔️ 
- [ ] A person has many rentals ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions

- [OPTIONAL] I think it would be better to add more description about the project so everyone can know what the project is about 😁 😁 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @peter-akhigbe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ Good work making the changes requested 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**